src/PlusVolumeReconstruction/vtkPlusVolumeReconstructor.cxx

```diff
@@ -883,3 +902,55 @@ void vtkPlusVolumeReconstructor::SetFanAnglesAutoDetectFilterRadiusPixel(int rad
 {
   this->FanAngleDetector->SetFilterRadiusPixel(radiusPixel);
 }
+
+//----------------------------------------------------------------------------
+void vtkPlusVolumeReconstructor::SetImportanceMaskFilename(std::string filename)
+{
+  if (this->ImportanceMaskFilename != filename)
+  {
+    this->ImportanceMaskFilename = filename;
+    
+    bool useVTK = true;
+    if (useVTK) //do not drag in ITK as a dependency
+    {
+      vtkSmartPointer<vtkPNGReader> reader = vtkSmartPointer<vtkPNGReader>::New();
```

Comment:
Is the CMake currently set up to link to ITK? If so, go for it. If not, maybe remove the ITK section.